strr-strata-web/app/stores/strataApplication.ts

```diff
@@ -27,7 +26,7 @@ export const useStrrStrataApplicationStore = defineStore('strr/strataApplication
 
   const validateStrataConfirmation = (returnBool = false): MultiFormValidationResult | boolean => {
     const result = validateSchemaAgainstState(
-      getConfirmationSchema(), platformConfirmation, 'platform-confirmation-form'
+      confirmationSchema, confirmation, 'platform-confirmation-form'
```

Comment:
Should we update this id to 'strata-confirmation-form' just to avoid confusion?

Reply:
yes definitely, I will update

Reply:
Cool, other than looks good! 🎉